frontend/dispatcher.lua

```diff
@@ -1,3 +1,30 @@
+--[[--
+Dispatcher module
+
+To add a new action an entry must be added to `settingsList` & `dispatcher_menu_order`
+This can also be done at runtime via `registerAction`
+    Dispatcher:registerAction("helloworld_action", {category="none", event="HelloWorld", title=_("Hello World"), filemanager=true,})
```

Comment:
Something more like this (a description, "Dispatcher module" is semi-meaningless):
```suggestion
--[[--
This module is responsible for dispatching events.

To add a new action an entry must be added to `settingsList` & `dispatcher_menu_order`
This can also be done at runtime via @{registerAction}().

    Dispatcher:registerAction("helloworld_action", {category="none", event="HelloWorld", title=_("Hello World"), filemanager=true,})

`settingsList` contains the list of dispatchable settings. Each setting contains:
```

Reply:
The newlines are for us; they don't affect rendering.